getndazn/dazn-lambda-powertools
# build(deps): install all deps

## What did you implement:

When running `lerna bootstrap` all dependencies should be installed in packages. However, due to a bug listed: https://github.com/lerna/lerna/issues/1457, this doesn't currently happen. Due to this, package-lock files for all packages are not fully representative of all deps and dev deps.

As a workaround, update from using `lerna bootstrap` to performing the commands as 2 commands manually.

This will allow package-lock files to be in sync as well as allowing dependabot automated updates to not fail.

All package-lock files have also been updated as a result.


## How did you implement it:

Update the `install` npm script to run `lerna exec npm ci && lerna link`

## How can we verify it:

Run `npm i` and ensure all packages have dependencies install and symlinked with package-lock files also updated with all dependencies.

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
